services/execution_layer.go

```diff
@@ -34,14 +36,60 @@ func latestBlockUpdater(wg *sync.WaitGroup) {
 	}
 }
 
-// LatestEth1BlockNumber will return the latest epoch
+// LatestEth1BlockNumber will return most recent eth1 block number
 func LatestEth1BlockNumber() uint64 {
-	cacheKey := fmt.Sprintf("%d:frontend:latestEth1BlockNumber", utils.Config.Chain.Config.DepositChainID)
+	cacheKey := fmt.Sprintf("%d:frontend:%s", utils.Config.Chain.Config.DepositChainID, latestBlockNumberCacheKey)
 
 	if wanted, err := cache.TieredCache.GetUint64WithLocalTimeout(cacheKey, time.Second*5); err == nil {
 		return wanted
 	} else {
-		logger.Errorf("error retrieving latestEth1BlockNumber from cache: %v", err)
+		logger.Errorf("error retrieving %s from cache: %v", latestBlockNumberCacheKey, err)
 	}
 	return 0
 }
+
+const latestBlockHashRootCacheKey = "latestEth1BlockRootHash"
+
+// headBlockRootHashUpdater updates the hash of the current chain head block
+func headBlockRootHashUpdater(wg *sync.WaitGroup) {
+	firstRun := true
+
+	for {
+		blockRootHash := []byte{}
+		err := db.ReaderDb.Get(&blockRootHash, `
+		SELECT blockroot
+		FROM blocks
+		WHERE status = '1'
+		ORDER BY slot DESC
```

Comment:
There is no index for just `slot` we should use one, or proof that a "multi-index" will cover this case.

Reply:
Similar to my answer here (https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2401#discussion_r1291084623), the already existing index `idx_blocks_status_slot_epoch` covers this as its second part is `slot`.
So this query ONLY uses `idx_blocks_status_slot_epoch` for both status and slot which works as the composite index has both columns as first and second part.

Reply:
good